src/app/pages/userslist/userslist.component.css

```diff
@@ -0,0 +1,37 @@
+#header {
+  display: inline;
+  align-content: center;
+}
+
+.separator {
+  margin: 10px;
+}
+
+#filter-bar {
+  margin: 10px;
+  display: flex;
+  justify-content: left;
+  justify-items: left;
+  padding: 0 20px 20px;
+}
+
+#filter-bar * {
+  margin: 0 0 10px 10px;
+}
+
+#filter-bar mat-card-footer{
+  display: flex;
+  justify-content: right;
+  justify-self: right;
+}
+
+#paging-bar {
+  display: flex;
+  justify-content: right;
+  justify-items: right;
+  margin: 10px;
+}
+
+#paging-bar button {
+  margin: 5px;
+}
```

Comment:
Always better to use `classes` instead of `ids` in CSS. (ids in special cases used)